CHANGELOG.md

```diff
@@ -20,6 +20,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - CHANGELOG.md @adieuadieu
 - LICENSE file (MIT) @adieuadieu
 - `implicitWait`: By default wait for `.press` and `.click` for a cleaner workflow
+- `.setUseragent()` is added to customize the userAgent you want to sent with your browser request 
```

Comment:
nitpick: elsewhere it's `setUserAgent` (capital A). Other than that looks good to me!